misk/src/test/kotlin/misk/config/ConfigTest.kt

```diff
@@ -3,16 +3,17 @@ package misk.config
 import com.google.common.truth.Truth.assertThat
 import misk.environment.Environment.TESTING
 import misk.environment.EnvironmentModule
-import misk.testing.MiskTestRule
+import misk.testing.MiskTest
+import misk.testing.ModuleProvider
+import misk.testing.Modules
 import misk.web.WebConfig
-import org.junit.Rule
-import org.junit.Test
+import org.junit.jupiter.api.Test
 import javax.inject.Inject
 import javax.inject.Named
 
+@MiskTest
```

Comment:
Should this be renamed to ActionTest?

Reply:
Could do. Will look at it in another PR.

Reply:
Done in this PR since I had to rebase anyway.